tools/build/build.js

```diff
@@ -34,6 +34,10 @@ export const CiParameter = new Juke.Parameter({
   type: 'boolean',
 });
 
+export const ErrorParameter = new Juke.Parameter({
+  type: 'boolean',
+})
+
```

Comment:
You should make a more generic parameter, e.g.

```ts
export const WarningParameter = new Juke.Parameter({
  type: 'string[]',
  alias: 'W',
});
```

which could then be used as `-Werror`, and you can check whether it has an error mode with:

```ts
get(WarningParameter).includes('error')
```

The benefit of doing that is:

- **Extensibility**, because later you might want to not treat certain warnings as errors or silence some of them.
- **Similarity with C compilers**. Compilers used `-Werror` for decades, and everyone knows what this flag does.

Reply:
Done.